app/models/ext_management_system.rb

```diff
@@ -422,6 +422,47 @@ def enable!
     update!(:enabled => true)
   end
 
+  def self.destroy_queue(ids)
+    ids = ids.to_miq_a
+    _log.info("Queuing destroy of #{name} with the following ids: #{ids.inspect}")
+    ids.each do |id|
```

Comment:
well we would have to manage that where clause since in each iteration some ids(providers) would be deleted and some would have to try again later so im not sure its worth it